src/index.js

```diff
@@ -516,6 +517,19 @@ class State extends EventEmitter {
 	/**
 	 * @private
 	 */
+	_channelMessage = (channel, message) => {
+		/**
+		 * This event is emitted when a message is sent to a mira.channel object
+		 * @event State.channel_message_received
+		 * @param {String} channel The name of the channel where the message was received
+		 * @param {Number|String|Array<Number|String>|Object} message The message received from Max
```

Comment:
should read
 @param {Number|String|Array<Number>|Array<String>|Object}

right?